.github/workflows/npm-publish-to-myget.yml

```diff
@@ -27,7 +27,7 @@ on:
     secrets:
 
       myget_api_key:
-        description: Please pass in the 'secrets.MYGET_DEPLOY_API_KEY_SECRET' value.
+        description: The secret API key needed in order to access the MyGet API.  These are formatted as GUIDs.
         required: true
```

Comment:
Okay to be required here?

Reply:
Yes, if we're down in this part of the call stack then we're definitely going to push to MyGet and we must have the key value.